filament/backend/src/opengl/OpenGLDriver.cpp

```diff
@@ -106,7 +106,8 @@
 
 #if DEBUG_MARKER_LEVEL > DEBUG_MARKER_NONE
 #   define DEBUG_MARKER() \
-        DebugMarker _debug_marker(*this, __func__);
+        const std::string _debug_marker_name = std::string("backend::") + __func__; \
+        DebugMarker _debug_marker(*this, _debug_marker_name.c_str());
```

Comment:
I don't what std::string operations (which have heap allocation) when tracing.  We need a solution that only relies on the preprocessor.

Reply:
I changed the GL implementation to use a stack-based char array instead of a std::string. Is that acceptable?
If yes, I'll go ahead and change the Vulkan backend too.
If not, I will just hardcore the marker names with litterals.